src/test/java/teammates/test/cases/logic/EmailGeneratorTest.java

```diff
@@ -265,6 +266,93 @@ public void testGenerateFeedbackSessionEmails() throws IOException {
+        for (EmailWrapper email : emails) {
+            if (email.getRecipient().equals(student1.email)) {
+                verifyEmail(email, student1.email, subject,
+                        "/sessionOpeningEmailTestingSanitzationForStudent.html");
+                hasStudent1ReceivedEmailCorrectly = true;
+            } else if (email.getRecipient().equals(instructor1.email)) {
+                verifyEmail(email, instructor1.email, subject,
+                        "/sessionOpeningEmailTestingSanitizationForInstructor.html");
+                hasInstructor1ReceivedEmailCorrectly = true;
+            }
+        }
+        assertTrue(hasStudent1ReceivedEmailCorrectly && hasInstructor1ReceivedEmailCorrectly);
+
+        ______TS("feedback session closed alerts: sanitization required");
+
+        emails = new EmailGenerator().generateFeedbackSessionClosedEmails(session);
+        assertEquals(2, emails.size());
+
+        subject = String.format(EmailType.FEEDBACK_CLOSED.getSubject(),
+                course.getName(), session.getFeedbackSessionName());
+
+        hasStudent1ReceivedEmailCorrectly = false;
+        hasInstructor1ReceivedEmailCorrectly = false;
+        for (EmailWrapper email : emails) {
+            if (email.getRecipient().equals(student1.email)) {
+                verifyEmail(email, student1.email, subject,
+                        "/sessionClosedEmailTestingSanitizationForStudent.html");
+                hasStudent1ReceivedEmailCorrectly = true;
+            } else if (email.getRecipient().equals(instructor1.email)) {
+                verifyEmail(email, instructor1.email, subject,
+                        "/sessionClosedEmailTestingSanitizationForInstructor.html");
+                hasInstructor1ReceivedEmailCorrectly = true;
+            }
+        }
+        assertTrue(hasStudent1ReceivedEmailCorrectly);
+        assertTrue(hasInstructor1ReceivedEmailCorrectly);
+
+        ______TS("feedback sessions summary of course email: sanitization required");
+
+        EmailWrapper email = new EmailGenerator().generateFeedbackSessionSummaryOfCourse(session.getCourseId(), student1);
+        subject = String.format(EmailType.STUDENT_EMAIL_CHANGED.getSubject(), course.getName(), course.getId());
+        hasStudent1ReceivedEmailCorrectly = false;
+
+        if (email.getRecipient().equals(student1.email)) {
+            verifyEmail(email, student1.email, subject,
+                    "/summaryOfFeedbackSessionsOfCourseEmailTestingSanitizationForStudent.html");
+            hasStudent1ReceivedEmailCorrectly = true;
+        }
+
+        assertTrue(hasStudent1ReceivedEmailCorrectly);
+
+        ______TS("feedback session submission email: sanitization required");
+
+        Calendar time = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        time.set(Calendar.DATE, 4);
+        time.set(Calendar.MONTH, 8);
+        time.set(Calendar.HOUR_OF_DAY, 5);
+        time.set(Calendar.MINUTE, 30);
+        time.set(Calendar.YEAR, 2016);
```

Comment:
This code block can be extracted somewhere appropriate as a util method? e.g. 
`Utils.getTime(2016, 4, 8, 5, 30)`